tests/components/core/test_cond.py

```diff
@@ -57,15 +58,23 @@ def test_validate_cond(cond_state: BaseState):
 
     [true_value_text] = true_value["children"]
     assert true_value_text["name"] == "RadixThemesText"
-    assert true_value_text["children"][0]["contents"] == "{`cond is True`}"
+    assert true_value_text["children"][0]["contents"] in (
+        "{`cond is True`}",
+        '{"cond is True"}',
+        "{'cond is True'}",
```

Comment:
is the test non-deterministic? seems like it would only ever be one of these right?